tests/integration/components/resources-list/edit-resource/component-test.ts

```diff
@@ -0,0 +1,132 @@
+        this.store = this.owner.lookup('service:store');
+    });
+
+    test('it renders when !@resource', async function(this: EditResourceTestContext, assert) {
+        const mirageRegistration = server.create('registration', 'currentUserAdmin');
+        assert.equal(mirageRegistration.resources.length, 0, 'registration does not have any resource');
+        this.registration = await this.store.findRecord('registration', mirageRegistration.id);
+        await render(hbs`
+            <ResourcesList::EditResource @registration={{this.registration}} as |modal| >
+                <Button
+                    data-test-open
+                    {{on 'click' modal.open}}
+                >
+                </Button>
+            </ResourcesList::EditResource>
+        `);
+        await untrackedClicked('[data-test-open]');
+        mirageRegistration.reload();
+        assert.equal(mirageRegistration.resources.length, 1, 'registration has exactly one resource');
+        await click('[data-test-preview-button]');
+        assert.dom('[data-test-validation-errors="pid"]').exists('DOI validation error message exists');
+        assert.dom('[data-test-validation-errors="pid"]').hasText(
+            this.intl.t(
+                'validationErrors.invalid',
+                {
+                    description: this.intl.t('osf-components.resources-list.edit_resource.doi'),
+                },
+            ),
+            'DOI validation error msg has correct language',
+        );
+        assert.dom('[data-test-validation-errors="resourceType"]').exists(
+            'Resource type validation error msage exists',
+        );
+        assert.dom('[data-test-validation-errors="resourceType"]').hasText(
+            this.intl.t(
+                'validationErrors.blank',
+            ),
+            'Resource type validation error msg has correct language',
+        );
+        await fillIn('[data-test-doi-field] > div > input', 'https://invalid_url');
+        await selectChoose('[data-test-resource-type-field]', 'Data');
+        assert.dom('[data-test-validation-errors="resourceType"]').doesNotExist();
+        await click('[data-test-preview-button]');
+        assert.dom('[data-test-validation-errors="pid"]').exists('DOI validation error message exists');
+        assert.dom('[data-test-validation-errors="pid"]').hasText(
+            this.intl.t(
+                'validationErrors.invalid',
+                {
+                    description: this.intl.t('osf-components.resources-list.edit_resource.doi'),
+                },
+            ),
+            'DOI validation error msg has correct language',
+        );
+        await fillIn('[data-test-doi-field] > div > input', 'https://doi.org/yeji');
+        await click('[data-test-preview-button]');
+        assert.dom('[data-test-resource-card-type]').exists('Shows preview');
+        const mirageResource = server.schema.resources.first();
```

Comment:
Could you check to make sure that all of the appropriate information is being shown in the preview?